### ISSUE TITLE
gatsby develop not working after updating to gatsby version 1.9.277

### ISSUE THREAD

**Issue by avocadoras** [gatsbyjs/gatsby#6637]
I updated my gatsby version after running `npm audit fix` and `gatsby serve` does not finish after calling it, only returns a blank screen on port 9000. 

MacOSX - 10.13.5 
gatsby cli - 1.1.58
node version - 9.10.1

However, `gatsby develop` does work so does `gatsby build`

**Comment by m-allanson** [comment]
Can you add the exact steps you used to reproduce this? I just tried using `gatsby new` to create a new site, but seem to have missed something as `gatsby serve` worked ok.

**Comment by avocadoras** [comment]
`gatsby new ttest
  cd ttest/
  ls
  npm audit fix
  gatsby develop
  gatsby serve`
![screenshot 2018-07-24 17 36 24](https://user-images.githubusercontent.com/36335687/43171588-1e939662-8f68-11e8-804b-edd16b7a6a45.png)
![screenshot 2018-07-24 17 36 54](https://user-images.githubusercontent.com/36335687/43171603-2d5fb68a-8f68-11e8-8183-3783e1f5675a.png)

**Comment by pawelsas** [comment]
Run build before serve?

**Comment by avocadoras** [comment]
@pawelsas yes, that was the issue .....my bad @m-allanson thanks for your patience
